docs/blockchain/contracts/nft-contract/nft-actions/rmfctofr.a.experimental.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'rmfctofr.a'
+order: 42
+
+---
+
+# rmfctofr.a
+
+Cancel the buy offer made on the Uniq factory.
+
+## Technical Behavior
+
+Note that `rmfctofr.a` is used to cancel the offer made by `mkfctofr.a` only.
+
+A buyer can cancel any of their offers at any time.
+
+Once the offer is expired, anyone can cancel the offer.
+
+When the offer is canceled, the offered price will be transferred from eosio.nftofr account back to buyer.
```

Comment:
Should be
When the offer is canceled, the offered price will be transferred from `eosio.nftofr` account back to buyer.

Reply:
updated